docs/setup/settings/options.md

```diff
@@ -16,7 +16,7 @@ Configure **Options** to set key default behaviors for the system. Click **`Edit
     |**Payment Details**|You can add custom messages and display details such as bank account information, simplifying the payment process. This is for display purposes only.|
     |**Payment Description**|Description that's displayed for payment methods such as PayPal and Stripe.|
     |**Smart Loading of Grid Tables**|You can choose this option to automatically load the tables while you scroll the page.|
-    |**Package Charge Notification Days**|You can choose the number of days you wish to receive the notification for package charging.|
+    |**Package Charge Notification Days**|You can set the package notification date. An e-mail will be sent the number of days before you wish to recharge the package.|

```

Comment:
You can set the package notification date, and an email will be sent the specified number of days before the package is scheduled to be recharged.

Reply:
done